shared/api/network_load_balancer.go

```diff
@@ -0,0 +1,166 @@
+package api
+
+import (
+	"net"
+	"strings"
+)
+
+// NetworkLoadBalancerBackendName represents a target backend name in a network load balancer
+//
+// swagger:model
+//
+// API extension: network_load_balancer.
+type NetworkLoadBalancerBackendName string
```

Comment:
What's the purpose for this custom type?

It's not one of those where we're dealing with a fixed set of options/enums and there are no functions attached to it, so it looks like it's just going to make our users' life potentially harder by having to cast between NetworkLoadBalancerBackendName and regular strings.

Reply:
I've removed it.